modules/retail/pages/retail-deploy-terminals-other.adoc

```diff
@@ -0,0 +1,173 @@
+. Build the image.
+. Prepare a USB device using this image, and boot the terminal.
+For more information about using USB devices to boot, see xref:retail:retail-deploy-terminals-other.adoc[].
+
+Alternatively, you can create the file and add it to the ``initrd``:
++
+----
+echo ./etc/sysconfig/network/ifcfg-wlan0 | cpio -H newc -o | gzip >> /srv/saltboot/boot/initrd.gz
+----
+
+Check that the terminals you want to deploy allow local booting.
+You can check this by editing the Saltboot formula in the {smr} {webui}.
+For more information about the Saltboot formula, see xref:salt:formula-saltboot.adoc[].
+
+
+
+=== Change Wireless Credentials
+
+After you have set the wireless credentials, you can change them as needed.
+The way to do this is different if you use one company-wide network, if you have each branch server on its own seperate network, or if you are using different wireless networks for booting and for regular system use.
+
+
+
+.Procedure: Changing Wireless Credentials for Single Network
+. Rebuild the boot image with updated credentials.
+. Recreate the bootable USB device based on the new boot image.
+. Boot terminal from new USB device.
+
+
+
+.Procedure: Changing Wireless Credentials for Multiple Networks
+. In the [path]``/srv/salt/`` directory, create a salt state called ``update-terminal-credentials.sls``, and copy the details from [path]``/etc/sysconfig/network/ifcfg-wlan0``.
+At a command prompt on the terminal:
++
+----
+/etc/sysconfig/network/ifcfg-wlan0
+  file.managed:
+   - contents: |
+        WIRELESS_ESSID=<wireless_network_name>
+        WIRELESS_WPA_PSK=<wireless_network_password>
+# regenerate initrd
+  cmd.run:
+  - name: 'mkinitrd'
+----
+. Apply the Salt state to the terminal:
++
+----
+salt <terminal_salt_name> state.apply update-terminal-credentials
+----
+
+[[NOTE]]
+====
+If you are using a separate network for the boot phase, the managed file might need to be renamed, or extended to [path]``/etc/sysconfig/network/initrd-ifcfg-wlan0``.
+====
+
+
+
+You can also instruct terminals to use a different set of wireless credentials during the boot process, to what they use during normal operation.
+This is done by using custom Salt states.
+
+First of all, you need to set the wireless details for normal operation.
+This will become the default settings.
+Then you can specify a second Salt state with the wireless details for use during the boot procedure.
+
+
+.Procedure: Using Different Wireless Credentials for Booting
+. Write a custom Salt state named ``/etc/sysconfig/network/ifcfg-wlan0`` containing the wireless details for normal operation.
+This Salt state is managed by Salt using ``saltboot_hook``, which is applied after the system image is deployed.
+If no other wireless credentials are supplied, these details will be used by default:
++
+----
+{% set root = salt['environ.get']('NEWROOT') %}
+{{ root }}/etc/sysconfig/network/ifcfg-wlan0:
+  file.managed:
+    - source:
+      - <salt://any/file>
```

Comment:
```suggestion
   - contents: |
        WIRELESS_ESSID=<wireless_network_name>
        WIRELESS_WPA_PSK=<wireless_network_password>
```

This will be better then vague source file.